deploy/tools/multus-test.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
```

Comment:
How about if we put this script under `tests/scripts`? Then separately we can discuss the long term home for it such as implementation as part of the krew plugin. I am just hesitating to support anything outside Rook if it's not in the krew plugin.